Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -1691,8 +1691,6 @@ QImage* Viewer_impl::takeSnapshot(Viewer *viewer, int quality, int background_co
     nbY++;
 
   QOpenGLFramebufferObject* fbo = new QOpenGLFramebufferObject(size, QOpenGLFramebufferObject::CombinedDepthStencil);
```

Comment:
Why is that `QOpenGLFramebufferObject fbo`  created on the heap (with new)? Create it on the stack, and its destructor will be called when the execution flow exits the scope.

Reply:
```c++
QOpenGLFramebufferObject fbo{size, QOpenGLFramebufferObject::CombinedDepthStencil};
```